docs/lms_config.ini

```diff
@@ -32,10 +31,17 @@ progress = SFD_check_progress.html
 
 learneraccountsettings = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-account-settings-page
 
-learnerdashboard = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-learner-dashboard
+learnerdashboard = sfd_dashboard_profile/SFD_dashboard_settings_profile.html#exploring-the-dashboard
+
+programs = SFD_enrolling.html#programs
 
 bookmarks = SFD_bookmarks.html
 
```

Comment:
hmm, and that's not universally true... course and learneraccountsettings seem to be indented by one, but not learnerdashboard. probably irrelevant!